lib/repo_tools.pm

```diff
@@ -558,4 +559,20 @@ sub verify_software {
     return '';
 }
 
+=head2 ensure_ca_certificates_suse_installed
+    ensure_ca_certificates_suse_installed();
+
+This functions checks if ca-certificates-suse is installed and if it is not it adds the repository and installs it.
+
+=cut
+
+sub ensure_ca_certificates_suse_installed {
+    return unless (is_sle || is_tumbleweed || is_leap);
+    if (script_run('rpm -qi ca-certificates-suse') == 1) {
+        zypper_call('ar --refresh http://download.suse.de/ibs/SUSE:/CA/' . generate_version() . '/SUSE:CA.repo');
+        zypper_call("in ca-certificates-suse");
+    }
```

Comment:
Just to show how an in-shell alternative could look like:

```suggestion
    assert_script_run 'rpm -qi ca-certificates-suse || zypper -n ar --refresh http://download.suse.de/ibs/SUSE:/CA/' . generate_version() . '/SUSE:CA.repo && zypper -n in ca-certificates-suse';
```

Reply:
Completely fine for me. I thought there is the advantage of `zypper_call()` which give us this big debug output if something fail.

Reply:
I prefer the `zypper_call` version, it's more readable IMO and doesn't call `generate_version` unless necessary.

Reply:
sure, valid reasons to keep the original version :)